msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md

```diff
@@ -31,7 +31,10 @@ Next, you can retrieve mentions using the `entities` object and add mentions to
 
 ## Work with mentions
 
-Every message to your bot from a group or channel contains an @mention with its name in the message text. Ensure that your message parsing handles @mention. Your bot can also retrieve other users mentioned in a message and add mentions to any messages it sends.
+> [!NOTE]
```

Comment:
We can remove the NOTE dialog here. It's already called out in the first section, above